From 0f364b57dc7c7e41e22d5bc6407b5c2f603694fa Mon Sep 17 00:00:00 2001
From: Behdad Esfahbod <behdad@behdad.org>
Date: Thu, 9 Jul 2026 12:09:01 -0600
Subject: [PATCH] Ignore -Wunused-template

Fixes https://github.com/harfbuzz/harfbuzz/issues/6059

This is squashed from upstream commits
5e887d36722abfef1c02a11ab15a369d7745027e and
b011381764f87be3cdd57b9a3205a51064016bba.
---
 src/hb.hh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/hb.hh b/src/hb.hh
index 7e2837f99..5d7c45c79 100644
--- a/src/hb.hh
+++ b/src/hb.hh
@@ -151,6 +151,7 @@
 #pragma GCC diagnostic ignored "-Wrange-loop-analysis" // https://github.com/harfbuzz/harfbuzz/issues/2834
 #pragma GCC diagnostic ignored "-Wstrict-aliasing"
 #pragma GCC diagnostic ignored "-Wtype-limits"
+#pragma GCC diagnostic ignored "-Wunused-template"
 #pragma GCC diagnostic ignored "-Wc++11-compat" // only gcc raises it
 #endif
 
-- 
2.43.0

